src/components/groups/PlayerList.tsx

```diff
@@ -80,11 +79,15 @@ const PlayerList = (props: Props) => {
   const formattedPlayers = players.map((p) => {
     const signup = playerStatus.get(p.username.toLowerCase());
     let status;
+    let end = new Date(0);
 
     if (signup == null) {
       status = '🚫';
     } else {
       const { startsAt, endsAt, court } = signup;
+      if (endsAt) {
```

Comment:
when a user is on a reserved court, endsAt will be null and they will end up sorting with the users that need to be signed up. I addressed this in the past by setting the sort order to be a very large number.